configs/daxpy/run-accel.py

```diff
@@ -177,7 +177,7 @@
     if exit_event.getCause() == "user interrupt received":
         print "User interrupt. Exiting"
         break
-    elif exit_event.getCause() == "target called exit()":
+    elif exit_event.getCause() == "simulate() limit reached":
```

Comment:
This actually isn't right. The target should call exit at the end of execution. There must be a bug in the user-mode app.